### ISSUE TITLE
UUID in WICG

### ISSUE THREAD

**Issue by bcoe** [WICG/uuid#1]
:wave: @broofa @ctavan.

After discussions with various folks, and the help of @domenic who's been helping guide me through the process, I've started drafting up a proposal for the [Web Incubator Community Group](https://wicg.io/).

I just pushed some initial rough work, which you can find in _index.html_, and at [bcoe.github.io/uuid](https://bcoe.github.io/uuid/)

## First off, an apology

I hope you can forgive me that we stalled in the standardization process, it was a combination of:

1. it becoming obvious that we were trying to standardize UUID at the wrong layer.
2. my getting buried in a combination of work, and dealing poorly with the mess that 2020 has been (_turns out being locked inside for 10 months isn't great for my anxiety_).

## Renewed interest/support for our work

@jasnell just landed a [WebCrypto](https://www.w3.org/TR/WebCryptoAPI/) implementation in Node@15, this eliminates one of the concerns that had come up about standardizing UUID in w3c (_the fact that Node did not have the same cryptography primitives as the browser_).

@jasnell and I would both love to add UUID to Node.js, but I think there would be value in fleshing out an API surface that could potentially work both for Node.js and other platforms.

----

_tldr; what do folks say we try to dust this work off, and go through the WICG process?_

**Comment by bcoe** [comment]
We've been successfully moving forward with the incubator process.
